### ISSUE TITLE
separate share component [suggestion]

### ISSUE THREAD

**Issue by acornejo** [shoutem/ui#128]
I am not sure social sharing functionality fits well inside a ui toolkit. If it were not included its something that could be implemented fairly easily on top of the existing ui toolkit and as such the benefits of baking it in are low (IMHO).

On the other hand, dropping this feature would mean less things to mantain and one less external dependency. In particular the only other native dependencies of this ui tookit are the fonts and linear gradient packages which have fairly widespread use and very well maintained, especially when compared to the share packet which hasn't seen a commit in months.

PS: What prompts this suggestion is that I ran into problems with the shoutem ui tookit with react-native 0.40, and it all boils down to this dependency.

**Comment by SoHotSoup** [comment]
@acornejo you are right, we've been talking about reducing native dependencies, there are two solutions for this issue, one is to implement RN's `Share`  and other to remove share helper from toolkit at all. We will take care of it in next few days.
